src/client/app/translations/data.ts

```diff
@@ -380,6 +380,8 @@ const LocaleTranslationData = {
 		"options": "Options",
 		"page.choice.login": "Page choices & login",
 		"page.choice.logout": "Page choices & logout",
+		"page.user.refresh.directions": "If clicking the 'Return to Dashboard button does not work then please click the button below to restart your OED session",
```

Comment:
I think it should be ``\"Return to Dashboard\"`` to get double quote and mirror how it is done in other places. This applies to all languages.